packages/commonwealth/client/scripts/views/modals/SessionRevalidationModal/SessionRevalidationModal.tsx

```diff
@@ -19,6 +19,8 @@ import TerraWalletConnectWebWalletController from 'controllers/app/webWallets/te
 import { X } from '@phosphor-icons/react';
 import { openConfirmation } from 'views/modals/confirmation_modal';
 import { setActiveAccount } from 'controllers/app/login';
+import IWebWallet from 'client/scripts/models/IWebWallet';
```

Comment:
Please do not use `client/scripts/` prefixed imports, per our [Code Requirements](https://github.com/hicommonwealth/commonwealth/blob/master/wiki/Code-Requirements.md)